enterprise/frontend/src/embedding-sdk/store/auth/auth.ts

```diff
@@ -96,13 +105,19 @@ export const initAuth = createAsyncThunk(
 export const refreshTokenAsync = createAsyncThunk(
   "sdk/token/REFRESH_TOKEN",
   async (
-    url: MetabaseAuthConfig["metabaseInstanceUrl"],
+    {
+      metabaseInstanceUrl,
+      authMethod: authMethod,
```

Comment:
Should we use a shortened form of argument? I've seen this in many places in the PR.

```suggestion
      authMethod,
```